parlai/agents/seq2seq/seq2seq.py

```diff
@@ -645,6 +645,9 @@ def load(self, path):
         if not os.path.isfile(path + '.opt'):
             # backwards compatible to old models
             self.opt = self.override_opt(states['opt'])
+            # save .opt file to make compatible
```

Comment:
hmm im not sure this shutdown is being called all the time.. i think just put it in the load function (in the init)?

Reply:
it is called in the load function which is called in init

Reply:
(it just appears that it is being called in shutdown on github because they are hiding a bunch of lines)